code/solr/searchquerywriters/SolrSearchQueryWriter_Basic.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+class SolrSearchQueryWriter_Basic extends AbstractSearchQueryWriter
+{
+    /**
+     * @var SearchCriterion $searchCriterion
+     * @return string
+     */
+    public function generateQueryString(SearchCriterion $searchCriterion)
+    {
+        $qs = $this->getComparisonPolarity($searchCriterion->getComparison());
+        $qs .= '(';
+        $qs .= $searchCriterion->getTarget();
+        $qs .= $this->getComparisonConjunction();
+        $qs .= $searchCriterion->getQuoteValue($searchCriterion->getValue());
+        $qs .= ')';
```

Comment:
I feel like something like this might look a little neater:
```php
return sprintf(
    '%s(%s%s%s)',
    $this->getComparis...,
    $searchCrit...,
     $this->getCompar....
    $searchCriter.....
);
```

What do you think? It's probably personal preference 😆